### DISCUSSION TITLE
komoot.io - different API response depending on client?

### DISCUSSION THREAD

**Discussion by gjedeer** [komoot/photon#749] [Q&A] [answered]
I'm trying to debug why my app is displaying wrong search results and discovered that querying the same URL with Firefox gives correct result, while querying it with curl gives imprecise results.

Complete description is here: https://github.com/gjedeer/Acastus/issues/29#issuecomment-1719768930

Is this a signal I should shut down the app and remove it from f-droid? Because I don't want to impose unwanted load on the free service you're providing.

**Comment by otbutz** [comment, marked as answer]
This is likely caused by the presence of the `Accept-Language` header in your browser.

**Comment by gjedeer** [comment]
Thank you - adding it to curl fixed the problem - I will now add it to the Android application too. Thank you.
